tests/MongoDB.Driver.Legacy.Tests/Operations/BulkWriteOperationTests.cs

```diff
@@ -202,28 +201,48 @@ public void TestExecuteTwice(bool ordered)
             Assert.Throws<InvalidOperationException>(() => bulk.Execute());
         }
 
+        public static MongoClientSettings GetClientSettings()
```

Comment:
Remove this method. Nothing calls it.

Reply:
I didn't notice that it wasn't called by me anymore. Done